src/TributeRange.js

```diff
@@ -465,7 +473,7 @@ class TributeRange {
         div.textContent = element.value.substring(0, position)
 
         if (element.nodeName === 'INPUT') {
-            div.textContent = div.textContent.replace(/\s/g, ' ')
+            div.textContent = div.textContent.replace(/\s/g, ' ')
```

Comment:
What's the change here? It looks like it replaced a space with a space.

Reply:
Not sure how this came about, I'm guessing some Git/Windows whitespace issue.